mmf/trainers/core/training_loop.py

```diff
@@ -61,9 +55,6 @@ def run_training_epoch(self) -> None:
             # Seed the sampler in case if it is distributed
             self.dataset_loader.seed_sampler("train", self.current_epoch)
 
-            if self.current_epoch > self.max_epochs:
-                break
```

Comment:
I am using num_updates and max_updates to control the stopping point rather than using epoch. max_epochs is being converted to max_updates in the `_calculate_max_updates` function.